content/en/docs/reference/score-schema-reference.md

```diff
@@ -0,0 +1,348 @@
+---
+title: "Score schema reference"
+linkTitle: "Schema reference"
+weight: 10
+description: >
+    Reference implementation for the Score schema.
```

Comment:
Is this a "reference implementation"? Would suggest something like "Validate your Score file with the Score schema" or so?

Reply:
@rachfop yes true! Any description that makes sense to you works (also just "Score schema") - I was just wondering if we want to refer to it as a "reference implementation"